.gitmodules

```diff
@@ -55,3 +55,6 @@
 [submodule "src/rapidjson"]
 	path = src/rapidjson
 	url = https://github.com/ceph/rapidjson
+[submodule "src/brotli"]
+	path = src/brotli
+	url = https://github.com/google/brotli.git
```

Comment:
awesome, but i'd suggest add brotli using `ExternalProject_Add()` and download brotli on the fly, see also https://github.com/ceph/ceph/blob/master/src/os/CMakeLists.txt#L122